Installed launch script fails

**Describe the bug**
This commit: https://github.com/marian-m12l/studio/commit/3ed38da8381f896e6a335f101a4fa533423185df breaks the "installed" version - when launching the script studio_windows it fails with this message:
```
Error: Could not find or load main class io.vertx.core.Launcher
Caused by: java.lang.ClassNotFoundException: io.vertx.core.Launcher
```

**To Reproduce**
Steps to reproduce the behavior:
1. Sync the depot at tag 0.3.0
2. Cherry-pick 3ed38d
3. mvn clean install
4. launch the script studio_windows from the extracted dist archive

**Expected behavior**
The server should be launched and a browser open localhost

**Logs**
No logs - the server simply cannot be launched.

**Desktop and environment (please complete the following information):**
 - Windows
 - openjdk version "16.0.1" 2021-04-20
 - Apache Maven 3.8.1
 - STUdio application version: 0.3.0 works fine, 0.3.0 plus commit 3ed38d breaks

--- a/web-ui/src/main/resources/studio-windows.bat
+++ b/web-ui/src/main/resources/studio-windows.bat
@@ -1,6 +1,6 @@
 @echo off
 
-set STUDIO_PATH="%~dp0"
+set STUDIO_PATH=%~dp0
 set DOT_STUDIO="%UserProfile%\.studio"
 
 :: Make sure the .studio subdirectories exist
@@ -9,7 +9,7 @@ if not exist %DOT_STUDIO%\db\* mkdir %DOT_STUDIO%\db
 if not exist %DOT_STUDIO%\library\* mkdir %DOT_STUDIO%\library
 
 :: Copy agent and metadata JARs
-copy %STUDIO_PATH%\agent\studio-agent-${project.version}-jar-with-dependencies.jar %DOT_STUDIO%\agent\studio-agent.jar
-copy %STUDIO_PATH%\agent\studio-metadata-${project.version}-jar-with-dependencies.jar %DOT_STUDIO%\agent\studio-metadata.jar
+copy "%STUDIO_PATH%\agent\studio-agent-${project.version}-jar-with-dependencies.jar" %DOT_STUDIO%\agent\studio-agent.jar
+copy "%STUDIO_PATH%\agent\studio-metadata-${project.version}-jar-with-dependencies.jar" %DOT_STUDIO%\agent\studio-metadata.jar
 
-java -Dvertx.disableDnsResolver=true -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager -Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.Log4j2LogDelegateFactory -Dfile.encoding=UTF-8 -cp %STUDIO_PATH%/${project.build.finalName}.jar;%STUDIO_PATH%/lib/*;. io.vertx.core.Launcher run ${vertx.main.verticle}
+java -Dvertx.disableDnsResolver=true -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager -Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.Log4j2LogDelegateFactory -Dfile.encoding=UTF-8 -cp "%STUDIO_PATH%/${project.build.finalName}.jar";"%STUDIO_PATH%/lib/*";. io.vertx.core.Launcher run ${vertx.main.verticle}

